templates/pro.html

```diff
@@ -0,0 +1,566 @@
+{% extends "_base/base.html" %}
+{% set active_page = "enterprise-support" %}
+
+{% block title %}Ubuntu Proエンタープライズ向けサポート | Ubuntu{% endblock %}
+
+{% block meta_description %}Landscapeは、CanonicalのサービスプログラムであるUbuntu Proで提供されるシステム管理ツールです。Ubuntuマシンの大規模なデプロイをまとめて簡単に管理し、大規模なデスクトップ、サーバー、クラウドのインスタンスのネットワークのサポートにおいてコスト効率を大幅に{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1xmk7zYUzWILnbW3RxGtC9c9stYYEFtKBytETlIgyHYE/edit{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block outer_content %}
+<div class="p-strip is-shallow u-no-padding--bottom">
+  <section class="p-section--shallow">
+    <div class="row">
+      <div class="col-3">
+        {{ image(url="https://assets.ubuntu.com/v1/c2c13f9d-Canonical%20Ubuntu%20Pro.svg",
+                  alt="",
+                  width="185",
+                  height="48",
+                  hi_def=True,
+                  loading="lazy") | safe
+        }}
+      </div>
+      <div class="col-medium-6 col-9">        
+          <div class="p-section--shallow">
+            <h1>Ubuntu&nbsp;Pro</h1>
+          </div>
+          <p>オープンソースソフトウェアのセキュリティを守る包括的なサブスクリプション<br />個人利用は無償。</p>
+        <div class="row">
+          <hr class="is-muted" />
+        </div>
+        <a href="/contact-us/form?product=pro" class="p-button--positive">お問い合わせ</a>
+        <a href="/pro/subscribe"
+           class="p-button">お申し込み</a>
+      </div>
+    </div>
+    <div class="p-suru"></div>
+  </section>
+
+  <section class="p-section u-no-padding--top">
+    <div class="row">
+      <hr class="u-no-margin--bottom" />
+      <h2>
+        Ubuntu LTSにセキュリティとコンプライアンスを追加：
+        <br />
+        10+2年間にわたり25,000以上のパッケージに対応
+      </h2>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="u-no-margin--bottom is-muted" />
+      <div class="col">
+        <div class="row">
+          <h2 class="p-text--small-caps u-sv2">セキュリティパッチ：</h2>
```

Comment:
Rather than nest this in an additional row, it should be wrapped in a p-section--shallow. usv classes aren't supposed to be used to modify the space between elements inside of Vanilla components, doc [here](https://vanillaframework.io/docs/utilities/vertical-spacing)
```suggestion
        <div class="p-section--shallow">
          <h2 class="p-text--small-caps">セキュリティパッチ：</h2>
        </div>
```